### PR TITLE
Tabs: Fix scrolling in live update orientation

### PR DESCRIPTION
**Pull request by marker-dao**

### CODE REVIEW

**Review by EugeniyKiyashko — DISMISSED** [review]

File: packages/devextreme/js/ui/tabs.js [hunk 1]
```diff
@@ -346,13 +357,30 @@ const Tabs = CollectionWidget.inherit({
         return this._$wrapper;
     },
 
-    _renderScrollable: function() {
-        const $itemContainer = this.$element().wrapInner($('<div>').addClass(TABS_SCROLLABLE_CLASS)).children();
+    _getScrollableDirection() {
         const isVertical = this._isVertical();
-        const scrollableDirection = isVertical ? 'vertical' : 'horizontal';
```
Comment (AlexanderMoiseev) [inline packages/devextreme/js/ui/tabs.js:352]: [minor] why don't use this._isVertical() directly here? in this case isVertical const is not needed. and this const doesn't make the code more readable

Comment (marker-dao) [reply packages/devextreme/js/ui/tabs.js:352]: Because this way is better for debugging.

File: packages/devextreme/js/ui/tabs.js [hunk 2]
```diff
@@ -346,13 +357,30 @@ const Tabs = CollectionWidget.inherit({
...
+    _updateScrollableDirection() {
+        const scrollable = this.getScrollable();
+
+        if(!scrollable) {
+            this._renderScrolling();
+        } else {
+            const scrollableDirection = this._getScrollableDirection();
+
+            scrollable.option('direction', scrollableDirection);
+        }
```
Comment (AlexanderMoiseev) [inline packages/devextreme/js/ui/tabs.js]:
```suggestion
        if(scrollable) {
            scrollable.option('direction', this._getScrollableDirection());
        } else {
            this._renderScrolling();
        }
```
the point here is why use 'not' here )

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/tabs.tests.js [hunk 3]
```diff
@@ -434,6 +434,32 @@ QUnit.module('Horizontal scrolling', () => {
...
+        const $element = $('#scrollableTabs').dxTabs({
+            items: [{ text: 'item 1' }, { text: 'item 1' }, { text: 'item 1' }, { text: 'item 1' }],
+            scrollingEnabled: true,
+            scrollByContent: true,
+            width: 100,
+            orientation: 'horizontal',
+        });
+
+        const getScrollable = () => {
+            const $scrollable = $element.children('.' + SCROLLABLE_CLASS);
```
Comment (AlexanderMoiseev) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets/tabs.tests.js]:
```suggestion
            const $scrollable = $element.children( `.${SCROLLABLE_CLASS}`);
```

**Review by AlexanderMoiseev — DISMISSED** [review]

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/tabs.tests.js (hunk 3 above)
Comment (marker-dao) [reply packages/devextreme/testing/tests/DevExpress.ui.widgets/tabs.tests.js]: [20cb1ef422683b209d85f6e3e12c47fff990f2d2]

File: packages/devextreme/js/ui/tabs.js (hunk 2 above)
Comment (marker-dao) [reply packages/devextreme/js/ui/tabs.js]: [20cb1ef422683b209d85f6e3e12c47fff990f2d2]

**Review by EugeniyKiyashko — APPROVED** [review]

**Review by AlexanderMoiseev — APPROVED** [review]

**Pull request merged**